src/test/java/com/amazon/dlic/auth/ldap/LdapBackendTest.java

```diff
@@ -387,7 +385,7 @@ public void testLdapAuthorization() throws Exception {
         Assert.assertNotNull(user);
         Assert.assertEquals("cn=Michael Jackson,ou=people,o=TEST", user.getName());
         Assert.assertEquals(2, user.getRoles().size());
-        Assert.assertEquals("ceo", new ArrayList(new TreeSet(user.getRoles())).get(0));
+        Assert.assertTrue(user.getRoles().contains("ceo"));
```

Comment:
Nit: The hamcrest `contains` matcher will make a much better error output if it fails, only a couple characters difference
```suggestion
        Assert.assertThat(user.getRoles(), contains("ceo"));
```